sidebars.js

```diff
@@ -284,6 +284,7 @@ module.exports = {
           Tasks: [
             "reference/bpmn-processes/tasks",
             "reference/bpmn-processes/service-tasks/service-tasks",
+            "reference/bpmn-processes/user-tasks/user-tasks",
```

Comment:
We need to add this page also to the overview: `/docs/reference/bpmn-processes/tasks.md`